### ISSUE TITLE
How to control expression in the  edit_sequence.py?

### ISSUE THREAD

**Issue by megajh96** [TimoBolkart/voca#114]
Thanks for share of the great work!
I want to know how to control variance of expression from flame model. 
and I want to change smile obj 
so I found.. alter_sequence_shape within the edit_sequence.py,  the model.betas[300:400] this can change the expression. 
I changed parameters.. but I don't know mouth control.. 
please help and thank you

**Comment by TimoBolkart** [comment]
Indeed, changing betas 300-400 can change the expression but it will mess up the speech-related expressions. For expression control, one would need to disentangle the speech- and emotion-related expression to change only the emotional content while keeping the speech-related content unchanged. Unfortunately, we don't have any solution for this yet.

**Comment by megajh96** [comment]
Thank you for your response! 
now, I don't use speech.. umm I just want to controll expression like webdemo
[https://flame.is.tue.mpg.de/interactivemodelviewer.html](url)
![image](https://user-images.githubusercontent.com/125354847/221056601-ffa9ae74-e655-4bd5-be41-df8e7a994de1.png)
